.template/addons/svgeez/Gemfile.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+# SVG Sprite
+insert_into_file 'Gemfile', after: /gem 'bcrypt'.*\n/ do
+  <<~RUBY
+
+    # Svgeez
+    gem 'svgeez' # Gem for generating an SVG sprite from a folder of SVG icons.
```

Comment:
I think we only need the `svgeez` gem to generate the sprite file,  so we only need it during development  - so could we move it inside the `development`  group?

Reply:
As I see it is now ` after: /gem 'danger'.*`, I assume this one is resolved :)

Reply:
Yeah. Fixed in 624b044431f780bd3bdc693bc378ae891eea1471 🙏🏼